tools/tools.mk

```diff
@@ -55,6 +55,13 @@ else
 	curl -LsS https://github.com/gohugoio/hugo/releases/download/v$(HUGO_VERSION)/hugo_extended_$(HUGO_VERSION)_macOS-64bit.tar.gz | tar zxv -C $(TOOLS_DIR)/hugo
 endif
 
+# fio
```

Comment:
Update the comment to say that the installation is conditional? and that it does not install in $(TOOLS_DIR).
We'll also see if there are suggestions or comments upstream.